Backbone.API/DevicesDbContextSeed.cs

```diff
@@ -36,47 +42,20 @@ private async Task SeedApplicationUsers(DevicesDbContext context)
         if (await context.Users.AnyAsync())
             return;
 
-        var basicTier = await GetBasicTier(context);
-
-        var user = new ApplicationUser
-        {
-            SecurityStamp = Guid.NewGuid().ToString("D"),
-            UserName = "USRa",
-            NormalizedUserName = "USRA",
-            Device = new Device(new Identity("test",
-                IdentityAddress.Create(new byte[] { 1, 1, 1, 1, 1 }, "id1"),
-                new byte[] { 1, 1, 1, 1, 1 }, basicTier.Id, 1
-            )),
-            CreatedAt = SystemTime.UtcNow
-        };
-        user.PasswordHash = _passwordHasher.HashPassword(user, "a");
-        await context.Users.AddAsync(user);
-
-        user = new ApplicationUser
-        {
-            SecurityStamp = Guid.NewGuid().ToString("D"),
-            UserName = "USRb",
-            NormalizedUserName = "USRB",
-            Device = new Device(new Identity("test",
-                IdentityAddress.Create(new byte[] { 2, 2, 2, 2, 2 }, "id1"),
-                new byte[] { 2, 2, 2, 2, 2 }, basicTier.Id, 1
-            )),
-            CreatedAt = SystemTime.UtcNow
-        };
-        user.PasswordHash = _passwordHasher.HashPassword(user, "b");
-        await context.Users.AddAsync(user);
-
-        await context.SaveChangesAsync();
+        await _mediator.Send(new SeedTestUsersCommand());
     }
 
     private async Task SeedBasicTier(DevicesDbContext context)
     {
-        if (await context.Tiers.AnyAsync())
+        try
+        {
+            var basicTier = await GetBasicTier(context);
             return;
-
-        await context.Tiers.AddAsync(new Tier(TierName.Create(TierName.BASIC_DEFAULT_NAME).Value));
-
-        await context.SaveChangesAsync();
+        }
+        catch (Exception)
```

Comment:
Catching all exceptions is generally a bad idea. You should only catch the exception you expect (in this case it should be an ApplicationException, I think. And then you should even refine for the specific error code (in this case it should be `error.platform.recordNotFound`, if I remember i right).